proto/ptypes/syncagent.proto

```diff
@@ -28,6 +28,8 @@ service SyncAgentService {
   rpc SnapshotHash(SnapshotHashRequest) returns (google.protobuf.Empty) {}
   rpc SnapshotHashStatus(SnapshotHashStatusRequest) returns
     (SnapshotHashStatusResponse) {}
+  rpc SnapshotHashInProgressTasks(SnapshotHashInProgressTasksRequest) returns
```

Comment:
`SnapshotHashStatus` only returns single snapshot's status.
`SnapshotHashInProgressTasks` returns the maximum number of in-progress hashing tasks in the associated replicas' node. Longhorn-manager will check this value and determine if it can issue a new request.